lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,96 @@
-# Detailed Reference
+# Detailed Reference Guide
 
-?> Coming Soon
+!> Under Construction.
+
+## Platform Setup
+
+Before implementing the Lotame Lightning Tag on your site, there is setup needed to be completed inside the Lotame Platform as described in the [Pre-Implementation Setup Tasks](lightning-tag/implementation-setup-tasks.md)
```

Comment:
language wise "there is .. needed to be completed" sounds wrong.  perhaps "there is setup that needs to be completed" ?